.github/workflows/lint_and_test_cpp.yaml

```diff
@@ -147,9 +147,13 @@ jobs:
           ./run.sh build --test --skip_thirdparty -j $(nproc) -t release
           ccache -s
       - name: Pack Server
-        run: ./run.sh pack_server
+        run: |
+          ./run.sh pack_server
+          rm -rf pegasus-server-*
```

Comment:
How about submitting this update to the master branch?

Reply:
OK.